setup.cfg

```
@@ -9,7 +9,7 @@ name = ansible-lint
 url = https://github.com/ansible/ansible-lint
 project_urls =
   Bug Tracker = https://github.com/ansible/ansible-lint/issues
-  CI: GitHub = https://github.com/ansible/ansible-lint/actions
+  CI: GitHub = https://github.com/ansible/ansible-lint/actions?query=workflow%%3Agh+branch%%3Amaster+event%%3Apush
```

@ssbarnea I see that you escaped `%` as `%%` but I think Python 3 doesn't need that. There was some difference of behaviors wrt this in Python 2.

I prefer not to urlencode this at all because it'd look more straightforward and predictable on PyPI.

```suggestion
  CI: GitHub = https://github.com/ansible/ansible-lint/actions?query=workflow:gh+branch:master+event:push
```

Proof that it works well w/o escaping: https://github.com/ansible/pylibssh/blob/6a59f7b/setup.cfg#L11 + https://pypi.org/project/ansible-pylibssh/